common/app/experiments/Experiments.scala

```diff
@@ -37,3 +37,12 @@ object OfferHttp3
       sellByDate = LocalDate.of(2022, 12, 6),
       participationGroup = Perc0B,
     )
+
+object KeyEventsCarousel
+    extends Experiment(
+      name = "key-events-carousel",
+      description = "Displays key events inside a carousel",
```

Comment:
Is it worth mentioning that the key events carousel is for on a live blogs?

Reply:
sounds good